src/Nest/XPack/MachineLearning/Job/Results/Influence.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	[JsonObject]
+	public class Influence
```

Comment:
This PR is full of awesome xmldocs but this one is left out 😿

Reply:
Done.